### ISSUE TITLE
Import Packages from Server

### ISSUE THREAD

**Issue by MWFIAE** [Co0sh/BetonQuestUploader#1]
I just discovered this and I wanted exactly something like this! :+1: 

But it would be really cool, if one could also import things from the server, so that my quest writers could also "pull" the latest version from the server (made from another user) and make there own changes before commiting again. 

Offtopic:
Actually while writing this I'm realizing a git-implementation would be absolutely amazing :grimacing: But then again I'm not so sure if my users could handle something like this...

**Comment by Co0sh** [comment]
I'd rather implement importing directly in the editor. While exporting is already supported with a nice shortcut, importing is not. It wouldn't be that convenient in a separate plugin.

I though about Git too and it would be awesome, but I don't think anyone else would even use it. It can be a tiny bit overwhelming for non-programmers 😛

**Comment by MWFIAE** [comment]
Yeah, I thought the combination of editor + the plugin to actually retrieve the files from the server without that whole ftp/etc. hassle. :) 


"non-programmers", I think git is overwhelming for everybody, even programmers, if they are not already used to it^^

**Comment by Co0sh** [comment]
Nah, I got it after only two hundreds visits on stack overflow, it's not that bad.

**Comment by MWFIAE** [comment]
I honestly don't want to know how often I had to google this stuff... because it was my idea to use it, so everybody would just come to me if they had any problems and/or conflicts with it... good old svn was actually really good to us xD

**Comment by Co0sh** [comment]
My biggest issue were line endings. Like when I edit a single line in a file but the editor decides to quietly convert all LF to CRLF. Commit, push...

15k additions, 15k deletions.

Other than that, it's pretty simple.
